editors/vscode/package.json

```diff
@@ -144,7 +149,77 @@
                     "contextualTitle": "Slint Properties"
                 }
             ]
-        }
+        },
+        "walkthroughs": [
+            {
+                "id": "slint.welcomeWalkthrough",
+                "title": "Get started with Slint Development",
+                "description": "Dive into VSCode's rich Slint development experience",
+                "steps": [
+                    {
+                        "id": "setup_cpp",
+                        "title": "Set up a C++ development environment (optional)",
+                        "description": "You need a C++ development environment to write business logic in C++.\n[Install C++ Extensions](command:workbench.extensions.installExtension?%22ms-vscode.cpptools-extension-pack%22)",
+                        "media": {
+                            "svg": "static/assets/cpp-logo.svg",
+                            "altText": "C++ logo"
+                        }
+                    },
+                    {
+                        "id": "setup_rust",
+                        "title": "Set up a Rust development environment (optional)",
+                        "description": "You need a Rust development environment to write business logic in Rust.\n[Install Rust Analyzer](command:workbench.extensions.installExtension?%22rust-lang.rust-analyzer%22)",
+                        "media": {
+                            "svg": "static/assets/rust-logo.svg",
+                            "altText": "rust logo"
+                        }
+                    },
+                    {
+                        "id": "create_slint_file",
+                        "title": "Create a .slint File",
+                        "description": "[Open](command:toSide:workbench.action.files.openFile) or [create](command:toSide:workbench.action.files.newUntitledFile?%7B%22languageId%22%3A%22slint%22%7D) a `.slint` file. Be sure to save it with the \".slint\" extension, such as \"helloworld.slint\". \n[Create a .slint File](command:toSide:workbench.action.files.newUntitledFile?%7B%22languageId%22%3A%22slint%22%7D)",
```

Comment:
should we mention that we can also write Slint code in a slint! macro in rust code?  Don't have to be in this commit.